package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "lib-build": "ng build ngx-wig",
-    "start": "(ng build ngx-wig --watch &) && sleep 10 && ng serve",
+    "start": "(ng build ngx-wig --watch ) && sleep 10 && ng serve",
```

Comment:
I don't mind changing it but it's not related to current PR

Reply:
I can remove it, I changed it because it was giving error by running the command